pkgs/applications/science/math/mathematica/10.nix

```diff
@@ -2,6 +2,7 @@
 , coreutils
 , patchelf
 , requireFile
+, lang, meta, name, src, version
```

Comment:
nitpick, but also you decide: maybe keep the item-per-line formatting?

Reply:
It was because it already had more than one item per line. I have forced the same style for all of `10.nix1, `11.nix`, etc. I also grouped the arguments and sorted each group alphabetically.

Reply:
👍🏻  As I said: I do not have any strong opinions on that. Personally, my heuristic is to write something that `nixpkgs-fmt` wouldn't alter